### PR TITLE
chore: use helper to centralize info about schema-supported databases

### PR DESCRIPTION
**Pull request by maelle**

### CODE REVIEW

File: NAMESPACE [hunk 1]
```diff
@@ -319,6 +319,7 @@ export(rows_patch)
 export(rows_truncate)
 export(rows_update)
 export(rows_upsert)
+export(schema_supported_dbs)
```
[the remark this answers is not in the document]
Comment (maelle) [reply NAMESPACE:322]: It was to use it in the vignette. Would you find using `dm:::schema_supported_dbs()` in the vignette better?

File: tests/testthat/helper-skip.R [hunk 2]
```diff
@@ -23,6 +23,15 @@ skip_if_src_not <- function(...) {
   }
 }
```
Comment (maelle) [inline tests/testthat/helper-skip.R:25]: related question https://github.com/cynkra/dm/pull/1123#issuecomment-1160129644

File: NAMESPACE (hunk 1 above)
[the remark this answers is not in the document]
Comment (krlmlr) [reply NAMESPACE:322]: Yes, let's keep using `:::` for now in the vignette.

**Comment by krlmlr** [comment]
Sorry about the conflicts, should be easy to resolve though.

File: pkgdown/_pkgdown.yml [hunk 3]
```diff
@@ -15,6 +15,7 @@ reference:
   - is_dm
   - dm_get_con
   - dm_get_tables
+  - schema_supported_dbs
```
Comment (krlmlr) [inline pkgdown/_pkgdown.yml:18]:
```suggestion
```

**Review by krlmlr — APPROVED** [review]

**Comment by krlmlr** [comment]
Thanks!

**Pull request merged**